### ISSUE TITLE
Incorrect semi-supervised evaluation?

### ISSUE THREAD

**Issue by denkorzh** [akhilmathurs/orchestra#6]
https://github.com/akhilmathurs/orchestra/blob/24b77df734e4eff5eade82b1540f94649af6134a/semisup_eval.py#L37

Hi, 
According to the paper (Section 5, "Evaluation protocol" paragraph), 
>  For semi-supervised evaluation, we fine-tune the entire model using limited labelled data (1% or 10% labels).

However, backbone weights are never fine-tuned. Only running statistics of BatchNorm are updated on the entire dataset.

**Comment by EkdeepSLubana** [comment]
Thanks for flagging this! There's a good chance that while releasing the code, we mistakenly copied a line from the linear eval file and the comment above is an artifact of that. I'll check this out in detail right away.

**Comment by denkorzh** [comment]
resolved by [b6ae718](https://github.com/akhilmathurs/orchestra/commit/b6ae718ffbebdb275e9197f33a940ff5a352a6a4)
